metro-ai-suite/smart-intersection/README.md

```diff
@@ -0,0 +1,2 @@
+**Smart Intersection Sample Application**
+Please read [Overview](docs/user-guide/Overview.md) for architecture overview and [Getting Started](docs/user-guide/get-started.md) for instructions on getting started with the sample application.
```

Comment:
Please provide a full README here.

Reply:
documentation template: https://github.com/intel-innersource/applications.services.esh.intel-edge-services-document/blob/edge-ai/docs/_templates/edge-ai/sample-application/sample-application-guidelines.md

Reply:
Populated main README with Overview.md content (with paths fixed to `./docs/user-guide/*`)